packages/data-explorer-ui/src/components/common/Button/components/CallToActionButton/callToActionButton.stories.tsx

```diff
@@ -31,6 +32,6 @@ CallToActionButtonStory.args = {
 export const CallToActionButtonWithCustomComponentStory =
```

Comment:
Hi @imadx, let's leave `CallToActionButtonWithCustomComponentStory` as is, with the prop `ButtonElType` defined as `ButtonElType: "a"`. I think I will make another ticket to remove this story altogether, because we can override story arguments if we need to. If you could revert line `34` for me please, and I will create a new ticket to remove this story altogether.

Reply:
Let's revert the packages/data-explorer-ui/src/components/common/Button/components/CallToActionButton/callToActionButton.stories.tsx altogether!